platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongList.java

```diff
@@ -67,14 +90,6 @@ public abstract class LongList implements CASableLongIndex, Closeable {
     protected final int numLongsPerChunk;
     /** Size in bytes for each memory chunk to allocate */
     protected final int memoryChunkSize;
-    /**
-     * The current maximum index that can be stored. This is determined dynamically based on the
-     * actual indexes used. It will grow, but never shrinks. Ultimately it would be nice to have
-     * some way to shrink this based on knowledge other parts of the system have about how many
-     * hashes need to be stored, but in the real system, this isn't important because we don't
-     * shrink the state size that dramatically, and on a reboot, this would be reset anyway.
-     */
-    protected final AtomicLong maxIndexThatCanBeStored = new AtomicLong(-1);
     /** The number of longs contained in this LongList. */
     protected final AtomicLong size = new AtomicLong(0);
```

Comment:
Should the JavaDoc for this field be updated now for a case when minValidIndex is not 0?

Reply:
Sure, I'll update it.